packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -17,6 +18,49 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = (t: Theme
     fluid: {
       width: '100%',
     },
+    fab: {
+      borderRadius: 100, // big number for always rounded corners
+      shadowColor: globalTokens.shadow[8][1].color.substr(0, 7),
+      shadowOffset: {
+        width: globalTokens.shadow[8][1].x,
+        height: globalTokens.shadow[8][1].y,
+      },
+      shadowRadius: globalTokens.shadow[8][1].blur,
+      shadowOpacity: Number('0x' + globalTokens.shadow[8][1].color.substr(7, 2)) / 255.0,
```

Comment:
You could define this helper above (with the helpful inline comments)
```
const shadowStyleFromGlobalToken = (shadowToken: number) => {
  const shadow = globalTokens.shadow[shadowToken][1] // Grab the second shadow from the global token, as we only support displaying one shadow
  return {
    // iOS Shadow props
    shadowColor: shadow.color.substr(0, 7), // split out the color
    shadowOpacity: Number('0x' + shadow.color.substr(7, 2)) / 255.0, // split out the opacity
    shadowRadius: shadow.blur,
    shadowOffset: {
      width: shadow.x,
      height: shadow.y
    },
    // Android shadow props
    elevation: shadowToken 
  }
}
```

and then do:
```
fab: {
  ...shadowStyleFromGlobalToken(8),
  // rest of props
}

```